### PR TITLE
T/711 Allow to drag widget only with left mouse button

### PR DESCRIPTION
**Pull request by beatadelura**
## What is the purpose of this pull request?

Bug fix

## Does your PR contain necessary tests?

Yes

### This PR contains

- [X] Manual tests

## What changes did you make?

Allowing to drag widget only on left mouse button by detecting pressed button.

---

Closes: #711 

Blocked by #810

### CODE REVIEW

File: plugins/widget/plugin.js [hunk 1]
```diff
@@ -3219,6 +3220,11 @@
 	}
 
 	function onBlockWidgetDrag( evt ) {
+		// Allow to drag widget only with left mouse button (#711).
+		if ( typeof evt.data != 'undefined' && evt.data.$.button != 0 ) {
```
Comment (Comandeer) [inline plugins/widget/plugin.js]: This code doesn't work in IE8 as it uses different values for buttons (0 means "no button" and 1 means "left button"). We have at least two implementations of such logic inside editor, you could look at e.g. [`copyformatting` plugin](https://github.com/ckeditor/ckeditor-dev/blob/f7deea1b22b7a975ee02adff0546e2b338301ed4/plugins/copyformatting/plugin.js#L13-L31).

File: tests/plugins/widget/manual/dragwidget.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div id="editor">
```
Comment (Comandeer) [inline tests/plugins/widget/manual/dragwidget.html]: I'm missing unit test for this case. It should be rather easy to write it: just fire `mousedown` event with appropriate data (remember about correct button values for IE8!) and check if `dragstart` event occurs.

File: tests/plugins/widget/manual/dragwidget.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@bender-tags: widget, bug, 4.7.2, 711
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, wysiwygarea, toolbar, sourcearea, image2, contextmenu
+
+**Scenario:**
+
+1. Click `Image` on toolbar.
+2. Input image URL, set width to 100 and mark `Captioned image` checkbox.
```
Comment (Comandeer) [inline tests/plugins/widget/manual/dragwidget.md]: IMO test should already include such image. We have dedicated asset for inserting images in tests: `lena.jpg`.

**Comment by Comandeer** [comment]
Taking over this issue and PR! ☠️

**Comment by f1ames** [comment]
Changed target branch to `major` and rebased on it.

**Pull request merged**